vulkan_helpers/vulkan_application.h

```diff
@@ -259,14 +260,19 @@ class PipelineLayout {
           CreateDescriptorSetLayout(allocator, device, binding_list.bindings_, binding_list.flags_));
       raw_layouts.push_back(descriptor_set_layouts_.back());
     }
+
+    containers::vector<VkPushConstantRange> raw_push_ranges(ranges.begin(),
```

Comment:
Looks reasonable, I would prefer this to be called push_constant_ranges.

Reply:
Done